std/internal/math/biguintcore.d

```diff
@@ -45,20 +45,19 @@ else
 alias multibyteAdd = multibyteAddSub!('+');
 alias multibyteSub = multibyteAddSub!('-');
 
-
-private import core.cpuid;
 private import std.traits;
 private import std.range.primitives;
 public import std.ascii : LetterCase;
 
-shared static this()
-{
-    CACHELIMIT = core.cpuid.datacache[0].size*1024/2;
-}
-
 private:
 // Limits for when to switch between algorithms.
-immutable size_t CACHELIMIT;   // Half the size of the data cache.
+// Half the size of the data cache.
+@nogc nothrow pure size_t getCacheLimit()
+{
+    import core.cpuid;
+    return (cast(size_t function() @nogc nothrow pure)
+        (() => core.cpuid.datacache[0].size * 1024 / 2))();
```

Comment:
I'm taking a bold guess here that this won't have a big performance-wise impact (at least with LDC).